doc/how_to_release_copr.rst

```diff
@@ -10,6 +10,15 @@ good to know what's ahead.
 Keep amending this page if you find something not matching reality or expectations.
 
 
+Create warning banner about release
+-----------------------------------
+
+
+Use script in frontend's ``manage.py`` to announce an outage::
+
+    ./manage.py manage-warning-banner --outage_time "2022-12-31 13:00-16:00 UTC" --ticket 1234
```

Comment:
manage manager warning banner? This is nitpicking, but can we have just
`manage.py warning-banner` ?